### DISCUSSION TITLE
docs cleanup

### DISCUSSION THREAD

**Discussion by joernhees** [RDFLib/rdflib#1564] [Documentation]
it would be a good idea to deprecate / remove / update old docs. This is a meta-issue to collect out-of date stuff. Feel free to report via comments, or edit (if you can) the following list:

- [ ] http://rdfextras.readthedocs.io/en/latest/index.html
   - [ ] make sure examples are integrated in core docs
   - [ ] deprecate / remove
- [ ] main readme on https://github.com/RDFLib/rdflib
   - [ ] explain RDFLib eco-system: relations between all the projects, and their "state"
- [ ] main docs http://rdflib.readthedocs.io/en/stable/
   - [ ] improve mainpage (better overview, actual content)
   - [ ] maybe a more complete tutorial (walk through every major aspect?)
   - [ ] make sure stuff is up-to-date (easy install vs pip etc)
   - [ ] go through examples again

**Comment by iddan** [comment]
Maybe Dataset be encouraged as the entry object for data?
